src/components/timecard/select-timecard-period-type/SelectTimecardPeriodType.spec.jsx

```diff
@@ -7,19 +7,55 @@ import SelectTimecardPeriodType from './SelectTimecardPeriodType';
 const mockRegister = jest.fn();
 const handleSubmit = jest.fn();
 const errors = {};
+const mockResponse = {
+  data: {
+    meta: {
+      next: null,
+    },
+    items: [
+      {
+        name: 'Shift',
+      },
+      {
+        name: 'Scheduled rest day',
+      },
+      {
+        name: 'Non-working day',
+      },
+      {
+        name: 'On call',
+      },
+      {
+        name: 'Absence',
+      },
+      {
+        name: 'Training',
+      },
+      {
+        name: 'Overtime',
+      },
+    ],
+  },
+};
+const timePeriods = [
+  'Shift',
+  'Scheduled rest day',
+  'Non-working day',
+  'On call',
+  'Absence',
+  'Training',
+  'Overtime',
+];
 
-describe('SelectTimecardPeriodType', () => {
-  it('should render a radios component with the correct time periods', () => {
-    const timePeriods = [
-      'Shift',
-      'Scheduled rest day',
-      'Non-working day',
-      'On call',
-      'Absence',
-      'Training',
-      'Overtime',
-    ];
+jest.mock('../../../api/services/timecardService', () => ({
+  getTimePeriodTypes: () => mockResponse,
+}));
 
+describe('SelectTimecardPeriodType', () => {
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+  it('should render a radios component with the correct time periods', async () => {
```

Comment:
Shouldn't it also check that the value of the input is set too e.g. to 00000000-0000-0000-0000-000000000001

Reply:
As discussed, I lept ahead here, this is beyond the scope of this requirement/ticket. 
The ticket that tackles a different shift type (e.g. Scheduled Rest Day) will cover checking the input selected (e.g. SRD) has the correct matching value (e.g. ...00002).